### PR TITLE
Add "Capture Rate" statistic

### PR DESCRIPTION
**Pull request by realOneplustwo**
Statistic to show how good a player is at capturing the flag.

### CODE REVIEW

File: mods/ctf/ctf_stats/chat.lua [hunk 1]
```diff
@@ -46,6 +47,7 @@ local function return_as_chat_result(to, target)
 			"\nBounty kills: " .. stat.bounty_kills ..
 			" | Captures: " .. stat.captures ..
 			" | Attempts: " .. stat.attempts ..
+			"\nCapture rate: " .. math.floor(cr * 100) .. "%" ..
```
Comment (Panquesito7) [inline mods/ctf/ctf_stats/chat.lua:50]: Shouldn't this be like

```diff
- 			"\nCapture rate: " .. math.floor(cr * 100) .. "%" ..
+ 			"\nCapture rate: %" .. math.floor(cr * 100)
```

**Review by realOneplustwo — COMMENTED** [review]
Should I keep it at math.floor or should I change it to math.round?

File: mods/ctf/ctf_stats/chat.lua (hunk 1 above)
Comment (realOneplustwo) [reply mods/ctf/ctf_stats/chat.lua:50]: That would make it look like "Capture rate: %100", instead of "Capture rate: 100%", which is what I am going for.

Comment (Panquesito7) [reply mods/ctf/ctf_stats/chat.lua:50]: OH yeah...thought my suggestion was the other way round. 😄

**Comment by appgurueu** [comment]
> Should I keep it at math.floor or should I change it to math.round?

Lua doesn't have `math.round`, use `math.floor(number + 0.5)` if you want it.

**Comment by Panquesito7** [comment]
Can someone re-run the LuaCheck checks?

File: mods/ctf/ctf_stats/chat.lua [hunk 2]
```diff
@@ -37,23 +37,26 @@ local function return_as_chat_result(to, target)
 
 	if stat then
 		local kd = stat.kills
+		local cr = stat.captures / stat.attempts
 		if stat.deaths > 1 then
 			kd = kd / stat.deaths
 		end
 		result = result ..
-			minetest.colorize("#63d437", "Kills: ") ..
+      minetest.colorize("#63d437", "Kills: ") ..
```
Comment (LoneWolfHT) [inline mods/ctf/ctf_stats/chat.lua:45]: This should be double-indented past line 44's indentation (same for the other similar lines)

**Review by LoneWolfHT — CHANGES_REQUESTED** [review]

File: mods/ctf/ctf_stats/chat.lua (hunk 2 above)
Comment (realOneplustwo) [reply mods/ctf/ctf_stats/chat.lua]: Fixed.

**Pull request closed without merging**